api/src/main/java/com/sanction/lightning/models/FacebookPost.java

```diff
@@ -1,5 +1,235 @@
 package com.sanction.lightning.models;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+import com.restfb.Facebook;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
 public class FacebookPost {
 
+  @Facebook("id")
+  private String facebookId;
+
+  @Facebook("caption")
+  private String caption;
+
+  @Facebook("created_time")
+  private Date createdTime;
+
+  @Facebook("description")
+  private String description;
+
+  @Facebook("from")
+  private Object from;
```

Comment:
I don't like using `Object` at all. Can we create some classes to use instead of `Object` for all the properties in this class?